pom.xml

```diff
@@ -304,10 +305,14 @@
                                 <exclude>**/META-INF/persistence*.xsd</exclude>
                                 <exclude>**/pom.xml.versionsBackup</exclude>
                                 <exclude>ide-config/**</exclude>
+                                <exclude>**/*.mvel</exclude>
+                                <exclude>**/quarkus-extension.yaml</exclude>
```

Comment:
I think we do not want the license headers in the quarkus-extension.yaml files. WDYT?

Reply:
If it does not harm the quarkus processing I would say we should add to be consistent with other similar files (i.e. properties)

Reply:
OK, removed the line.